website/docs/guides/best-practices/how-we-mesh/mesh-1-intro.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Intro to dbt Mesh"
+description: Getting started with dbt Mesh patterns
+hoverSnippet: Learn how to get started with dbt Mesh
+---
+
+## What is dbt Mesh?
+
+Historically, building data teams has involved two extremes, building a centralized team or using embedded analysts. More recently, hub-and-spoke models have become popular as a way to balance the tradeoffs: using a centralized platform team _and_ embedded analysts, allowing embeds to develop domain expertise while the central team focuses on building a strong operational foundation. A major difficultly of this model though is managing the compplexity of dependencies, goverance, and workflows between all groups — creating friction in monorepos or complexity and silos in multi-repos. Ideally, you want to teams to be able to work independently, but also be able to collaborate; sharing data, code, and best practices. dbt Mesh provides the tooling for teams to finally achieve this.
```

Comment:
```suggestion
Historically, building data teams has involved two extremes, building a centralized team or using embedded analysts. More recently, hub-and-spoke models have become popular as a way to balance the tradeoffs: using a centralized platform team _and_ embedded analysts allows embeds to develop domain expertise while the central team focuses on building a strong operational foundation. A major difficultly of this model is managing the complexity of dependencies, governance, and workflows between all groups — attempting to manage this can often create friction in monorepos or complexity and silos in multiple repos. Ideally, teams should be able to work independently and also be able to collaborate; sharing data, code, and best practices. dbt Mesh provides the tooling for teams to finally achieve this.
```